draft-iab-covid19-workshop.md

```diff
@@ -240,11 +240,11 @@ to share their experiences. The meeting was held online given the
 on-going travel and contact restrictions at that time.
 
 COVID-19 has caused changes in user behavior, which in turn drove 
-change to Internet traffic. These changes appeared rather abruptly 
+change to Internet traffic. These changes in user behavior appeared rather abruptly 
 and were significant, in particular during the introduction of the
-initial quarantine and work-from-home arrangements. The changes relate
-to traffic volumes, location of traffic, as well as the types of
-traffic and applications used.
+initial quarantine and work-from-home arrangements. The changes to Internet traffic relate
+to volumes, location, types, as well as security and attack methodologies,
```

Comment:
While I understand the desire to mention security, I think it really does fit here. I think the first directly consequences of sending everybody home was a change in traffic patterns. Chances in attack behaviour and new security practices where a consequence of that (slightly later). Or what you actually means by security mythologies here?

Reply:
```suggestion
to traffic volumes, location of traffic, as well as shifts in the type of traffic
```